src/app/mail/mail-settings/mail-settings.component.html

```diff
@@ -347,14 +347,13 @@ <h5 class="ui-header-subtitle text-dark mb-0"><strong translate="settings.genera
                             <div class="btn-dropdown btn-dropdown-select">
                                 <div ngbDropdown class="dropdown-sm">
                                     <button ngbDropdownToggle class="dropdown-toggle" id="timeZoneDropdownMenuButton"
-                                            data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">UTC+1
-                                        (Italy, Germany)
+                                            data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                                        {{settings.timezone}}
                                     </button>
                                     <div ngbDropdownMenu aria-labelledby="timeZoneDropdownMenuButton">
-                                        <a class="dropdown-item">UTC+1 (Italy, Germany)</a>
-                                        <a class="dropdown-item">UTC+1 (Italy, Germany)</a>
-                                        <a class="dropdown-item">UTC+1 (Italy, Germany)</a>
-                                        <a class="dropdown-item">UTC+1 (Italy, Germany)</a>
+                                        <a class="dropdown-item"
+                                           *ngFor="let timezone of timezones"
+                                           (click)="settings.timezone=timezone.value; updateSettings()">{{timezone.value}}</a>
```

Comment:
I think its better to move this into a function instead of changing it inline.

Reply:
Actually there will be a lot of settings to update on this page, so we have a function which is called for each update, `updateSettings`, 
this settings object is kind of immutable, when we change it and call updateSettings, then they are updated and settings value in store is also update, and a new settings object is assigned to this one.

Reply:
Ok. Its fine then.